core/controllers/base.py

```diff
@@ -166,33 +166,32 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
             self.payload = None
         self.iframed = False
 
-        self.is_super_admin = user_services.is_current_user_super_admin()
+        auth_claims = user_services.get_auth_claims_from_request(request)
+        self.is_super_admin = auth_claims is not None and auth_claims.is_admin
```

Comment:
Same note for is_admin -- should be a method. Or alternatively: auth_claims.user_is_admin

Also, maybe we should call this super_admin instead of admin throughout, to prevent confusion with the "admin" role?

Reply:
Done.